src/dataDisplay/popover/popover.jsx

```diff
@@ -0,0 +1,131 @@
+import Classnames from 'classnames';
+import { get } from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { BEM_POPOVER } from '../../global/constants';
+import makeStyles from '../../styles/makeStyles';
+import ToolTip from '../tooltip';
+
+const propTypes = {
+    /**
+     * The arrow can be enabled/disabled
+     */
+    arrow: PropTypes.bool,
+    /**
+     * Additional classes.
+     */
+    className: PropTypes.string,
+    /**
+     * Content
+     */
+    content: PropTypes.string,
+    /**
+     * We can pass either a string or jsx elements
+     */
+    children: PropTypes.shape({}).isRequired,
+    /**
+     * Popper container styles
+     */
+    popperStyles: PropTypes.shape({}),
+    /**
+     * Style applied to the root container
+     */
+    style: PropTypes.shape({}),
+    /**
+     * We can define the max width for the popover
+     */
+    maxWidth: PropTypes.oneOfType([
+        PropTypes.number,
+        PropTypes.string,
+    ]),
+    /**
+     * We can define the width for the popover
+     */
+    width: PropTypes.oneOfType([
+        PropTypes.number,
+        PropTypes.string,
+    ]),
+};
+
+const defaultProps = {
+    arrow: true,
+    content: undefined,
+    className: undefined,
+    maxWidth: undefined,
+    style: undefined,
+    popperStyles: {},
+    width: 250,
+};
+
+const useStyles = makeStyles((theme) => ({
+    arrow: {
+        color: get(theme, 'palette.common.white'),
+    },
+    popper: ({ popperStyles }) => (popperStyles || {}),
+    root: {
+        width: 'fit-content',
+    },
+    tooltip: {
+        backgroundColor: get(theme, 'palette.background.primary'),
+        color: 'black',
+        padding: 22,
+        borderRadius: get(theme, 'shape.borderRadius.secondary'),
+        maxWidth: ({ maxWidth }) => maxWidth || 'none',
+        fontFamily: get(theme, 'typography.fontFamily'),
+        fontSize: get(theme, 'typography.fontSize'),
+        fontWeight: get(theme, 'typography.fontWeightRegular'),
+        width: ({ width }) => width,
+    },
+}));
+
+/**
+ * The Popover is similar to tooltips, it is a pop-up box that appears when the user
+ * hovers on an element
+ */
```

Comment:
Should say "... is similar to _a tooltip;_ ..." instead of "... similar to tooltips,...".  i.e. compare singular Popover to singular Tooltip.  Also, use a semicolon there instead of a comma, as it is separating two complete clauses/sentences.  Maybe also replace "on" with "over" and a period after "hover".

```JavaScript
/**
 * The Popover is similar to a tooltip; it is a pop-up box that appears when the user
 * hovers over an element.
 */
```

Reply:
done

Reply:
All but the ending period ;-)
All good.  I'll mark this resolved.